.circleci/config.yml

```diff
@@ -98,6 +98,11 @@ jobs:
           name: Run tests
           no_output_timeout: 20m
           command: |
+            CIRCLE_PR_BRANCH=$([[ -n ${CIRCLE_PR_NUMBER} ]] && (curl -s https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/pulls/${CIRCLE_PR_NUMBER} | jq -r '.head.ref') || "")
+            echo "CIRCLE_PR_BRANCH: ${CIRCLE_PR_BRANCH}"
+            if [[ $CIRCLE_BRANCH =~ ^i18n || $CIRCLE_PR_BRANCH =~ ^i18n ]]; then echo "Skipping: i18n branch"; exit 0; fi
+            if [[ $CIRCLE_BRANCH =~ ^docs || $CIRCLE_PR_BRANCH =~ ^docs ]]; then echo "Skipping: docs branch"; exit 0; fi
+            if [[ $CIRCLE_BRANCH =~ ^update-builder || $CIRCLE_PR_BRANCH =~ ^update-builder ]]; then echo "Skipping: update-builder branch"; exit 0; fi
```

Comment:
hmm I'm wondering if we should add a helper script in `devops/` for this logic for DRYness reasons/readability? what do you think? (it would take a list of prefixes of branches to skip) then we can call out to it in these run steps

Reply:
Yeah, good call. Will do.